components/LandingPage/sections/WelcomeToCre8ors.tsx

```diff
@@ -128,7 +134,12 @@ const WelcomeToCre8ors: FC<Props> = ({
           </SectionContent>
         </div>
         <div ref={inputRef} className="appear mb-6 md:mx-12 flex justify-center md:justify-start">
-          <div className="w-[350px] md:w-[416px]">
+          <div
+            className="w-[350px] md:w-[416px]"
+            style={{
+              width: isIphone ? "290px" : "",
+            }}
```

Comment:
rather than checking using media query, just add the iphone screen size to tailwind. https://tailwindcss.com/docs/screens

Reply:
Okay, I will customize it.

Reply:
yeah I like Sameer's idea 💡